config_manager.go

```diff
@@ -35,6 +37,16 @@ type ConfigFile struct {
 	ExcludePatterns *RegexCollection `yaml:"exclude_patterns"`
 }
 
+func (cf ConfigFile) printValidationWarnings() {
+	if len(cf.Files) == 0 {
+		log.Warningf("Validation: configuration file contains no files to watch")
+	}
+	// Warn, that port is not set, only if host is specified
+	if cf.Destination.Host != "" && cf.Destination.Port == 0 {
+		log.Warningf("Validation: destination port is not specified in configuration file")
```

Comment:
@troy It definitely bothers me that someone could accidentally miss the port off and we wouldn't tell them, but I agree that breaking existing configs sucks and has to take priority.

@featalion Sorry, I should have clarified that `514` is assigned to `DefaultPort` (which is a constant) in the same file as the code above. `DestPort()` seems to consider both the CLI and config file settings:

```
case cm.Flags.DestPort != 0: // CLI
    return cm.Flags.DestPort
case cm.Config.Destination.Port != 0: // Config file
    return cm.Config.Destination.Port 
```

Let me know if I am reading this wrong.